src/pages/api/sync.ts

```diff
@@ -0,0 +1,305 @@
+import { NowRequest, NowResponse } from "@now/node";
+import { google } from "googleapis";
+import { MemberUpsertArgs, PrismaClient } from "@prisma/client";
+import { range, zipObject } from "lodash-es";
+import to from "await-to-js";
+import { normalizeParam } from "utils";
+
+const prisma = new PrismaClient();
+
+const sheets = google.sheets({
+  version: "v4",
+  auth: new google.auth.GoogleAuth({
+    keyFile: "./.google-api-creds.json",
+    scopes: [
+      "https://www.googleapis.com/auth/drive",
+      "https://www.googleapis.com/auth/drive.file",
+      "https://www.googleapis.com/auth/spreadsheets",
+    ],
+  }),
+});
+
+// Some cells can contain multiple values separated by a `;`
+const splitMultiValueCell = (cellValue: string): string[] | undefined => {
+  const values = cellValue
+    .split(";")
+    .map((v) => v.trim())
+    .filter((v) => v.length > 0);
+
+  return values.length > 0 ? values : undefined;
+};
+
+export default async function sync(req: NowRequest, res: NowResponse) {
```

Comment:
I usually have these API handlers typed as [`NextApiRequest` and `NextApiResponse`](https://nextjs.org/docs/basic-features/typescript#api-routes) — is this just legacy or is there a substantive difference here?